geometry_msgs/Pose pose
float64[] jointAngles
---
float64[] bestSolution